src/css/styles.scss

```diff
@@ -1,6 +1,42 @@
 @import 'variables';
 @import 'mixins.scss';
 
+@mixin inputFields {
```

Comment:
Nice use of mixins!! This all looks really good and makes a lot of sense as those inputs and buttons are identical and being repeated across all cards!

Reply:
Good call. I didn't even think about the mixin file, but that's an easy fix.